.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+# Description
+
+Please include a summary of the change.
+
+## Fixes
+Please add related issues.
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+# Checklist
+
+- [ ] The pull request is for the branch develop
+- [ ] I changed the file [Usage](https://github.com/intelowlproject/IntelOwl/blob/master/docs/source/Usage.md) and [ReadMe](https://github.com/intelowlproject/IntelOwl/blob/master/README.md) if I added a new analyzer 
+- [ ] I have added tests in the [Tests](https://github.com/intelowlproject/IntelOwl/blob/master/tests) folder if I added a new analyzer 
+- [ ] The tests gave 0 errors
+- [ ] `Black` gave 0 errors
+- [ ] `Flake` gave 0 errors
+- [ ] I squashed the commits into a single one
```

Comment:
I would also add the new line I added in the docs recently: `In the Pull Request remember to provide some real world examples (screenshots and raw JSON results) of some successful executions of the analyzer to let us understand how it would work.`